getting-started/kubearmor_visibility.md

```diff
@@ -0,0 +1,128 @@
+# KubeArmor Visibility
+
+KubeArmor currently supports enabling visibility for containers/hosts
+Visibility for containers is enabled by default and can be accesed using `karmor` tool
+
+
+
+## Getting Container Visibility
+### Example : multiubuntu
+* Prerequisites
+
+ The following  example would require a K8s cluster , please follow [this](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#prerequisites) to setup one if you dont have access to one
```

Comment:
Best to use [grammarly](https://app.grammarly.com/) to fix the basic grammatical issues.